conf/aggregations.go

```diff
@@ -85,6 +91,38 @@ func ReadAggregations(file string) (Aggregations, error) {
 			}
 		}
 
+		writeBufferStr := s.ValueOf("writeBuffer")
```

Comment:
That just reminded me of something:
If we allow that to be specified as a time we'll need to enforce that this time is a multiple of the interval. This reminded me that the rollups have a different intervals than the first aggregate, so when the write buffer is instantiated for the rollups the reorder window should be divided by `raw interval / aggregate interval`